xapian-core/tests/termgentest.cc

```diff
@@ -735,7 +755,22 @@ static bool test_termgen1()
 	    } else if (strncmp(o, "some", 4) == 0) {
 		o += 4;
 		termgen.set_stemming_strategy(termgen.STEM_SOME);
-	    } else if (strncmp(o, "prefix=", 7) == 0) {
+	    } else if(strncmp(o, "stop=", 5) == 0) {
+               o += 7;
+               vector<string> x = {"the", "a", "an"};
```

Comment:
`vector<string>` is overkill here - the size is constant and known - `array<const char*, 3>` should work just as well.